needle/modules/configuration/mdm_assess.py

```diff
@@ -0,0 +1,131 @@
+from core.framework.module import BaseModule
+from core.device.device import Device
+from core.utils.menu import choose_from_list_data_protection, choose_from_list, choose_boolean
+from core.utils.utils import Utils
+import re, sys
+
+class Module(BaseModule):
+    meta = {
+        'name': 'MDM Assess',
+        'author': 'Oliver Simonnet (@MWRLabs)',
+        'description': 'Automated MDM Configuration Assessment tool.',
+        'options': (
+            ('template', True, True, 'Configuration template.[Plist|plutil-output]'),
```

Comment:
The configuration template would be a pre-created EffectiveUserConfiguration.plist file.
Alternately a user could use a template provided by needle (maybe a file stored within _templates?)

This would be an EffectiveUserSettings.plist file in JSON  format (output given by using plutil on the EffectiveUserSettings.plist file (a lot easier to manual alter))